books/free-programming-books-de.md

```diff
@@ -150,6 +150,7 @@
 * [JavaScript](http://www.peterkropff.de/site/javascript/javascript.htm) - Peter Kropff (Grundlagen, AJAX, DOM, OOP) (Online, PDF)
 * [JavaScript und AJAX](http://openbook.rheinwerk-verlag.de/javascript_ajax) - Christian Wenz (Online)
 * [Webseiten erstellen mit Javascript](http://www.highscore.de/javascript) - Boris Schäling (HTML)
+* [Bible of JS](https://shorturl.at/ax056) - Sheryians Coding School (JavaScript)
```

Comment:
```suggestion
* [Bible of JS](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbjVFU1JDVTlPS1hodnYzYWFORmpheExENjk4UXxBQ3Jtc0tuVERYNW84YUtUQ2oxUS1kX185NnBYb3g5Q0lzYUpzM2RCVVoxZUVydDRxeTBJOFptdjQ5NW16dUNiQlBWUmRRdWR6QjFfak90RVpuaE41TzBlQ21FRDFJLXVTMkE1MzJMbnMzS1VFRnNNOEdHQW1Odw&q=https%3A%2F%2Fsheryians.com%2Fdownload%2Fbibleofjs_by_sheryians&v=Aw_SmcpEOdY) - Harsh Sharma (PDF)
```

Reply:
not redirects either!

Reply:
Yeah because author provide only this link, that's why this link directly download the book in pdf format